Tests/APIClientTests.swift

```diff
@@ -10,27 +10,35 @@ import XCTest
 @testable import APIClient
 
 class APIClientTests: XCTestCase {
-    
+
+    let testURL = URL(string: "intrepid.io")!
+    var sut: APIClient!
```

Comment:
"system under test", which basically represents the main object that the unit tests file is focusing on.